src/pages/Widget/Widget.tsx

```diff
@@ -75,7 +76,7 @@ function Content({
       <h1 className="text-lg @4xl/widget:text-2xl text-center @4xl/widget:text-left mb-3 @4xl/widget:col-span-2">
         Accept donations from your website today!
       </h1>
-      <p className="text-center @4xl/widget:text-left text-sm @4xl/widget:text-base @4xl/widget:col-span-2 pb-2">
+      <p className="mb-8 text-center @4xl/widget:text-left text-sm @4xl/widget:text-base @4xl/widget:col-span-2 pb-2">
```

Comment:
add space so that header article isn't confused to belong to the left column
<img width="1470" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/e0018f31-4ac9-477c-b651-3895764872e5">

now 
<img width="1457" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/769aac53-fae2-4748-8e51-69178a5012f8">

Reply:
Much nicer!